workspaces/adr/package.json

```diff
@@ -48,6 +48,8 @@
   "resolutions": {
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "@types/unist": "^2.0.0",
+    "markdown-table": "^3.0.4",
```

Comment:
Can you clarify why these resolutions are needed? would this be required for consumers of this plugin in their Backstage apps?

Reply:
`@types/unist` is needed to pinpoint v2 instead of v3 for type inconsistency between `react-markdown` and other dependencies.

example error: https://github.com/backstage/community-plugins/actions/runs/19835900496/job/56833009521

I've reverted the `markdown-table` change because apparently it's not needed to make it work